### PR TITLE
Api docs updates and fixes [WIP]

### PR DESCRIPTION
**Pull request by catalinstoica**
Remaining tasks:

- [x] Check if we've all API parameters documented
- [x] Check all shell sample codes
- [x] Check all ruby sample codes
- [x] Check all python sample codes
- [ ] Check all java sample codes
- [x] Check all php sample codes

### CODE REVIEW

File: source/includes/app-publishers.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 > Don't forget to replace `{API_KEY}` with your actual API key.
 
 ```shell
-curl --compress -u "{API_KEY}:X" "https://api.appmonsta.com/v1/stores/android/publishers.json"
+curl --compress -u "{API_KEY}:X" "https://api.appmonsta.com/v1/stores/android/publishers.json?date={{ date }}"
```
Comment (lknix) [inline source/includes/app-publishers.md]: This one is optional, right?

Comment (lknix) [reply source/includes/app-publishers.md]: Err, is `date` a valid parameter for this call?

Comment (lknix) [reply source/includes/app-publishers.md]: Also there's a code typo (`$date`) in Java code:

```
 .queryString("date", $date)
```

Let's make sure we actually run code samples, not just fix them.

File: source/includes/rankings.md [hunk 2]
```diff
@@ -228,7 +228,7 @@ username = "{API_KEY}"  # Replace {API_KEY} with your API own key.
 password = "X"          # Password can be anything.
 
 # Request URL
-uri = URI("https://api.appmonsta.com/v1/stores/android/aggregate.json?date=#{date}&country=#{country_code}")
+uri = URI("https://api.appmonsta.com/v1/stores/android/rankings/aggregate.json?date=#{date}&country=#{country_code}")
```
Comment (lknix) [inline source/includes/rankings.md:248]: Please run code samples after you fix them: copy/paste - to make sure there are no errors.

File: source/includes/rankings.md [hunk 3]
```diff
@@ -165,6 +165,23 @@ curl_close($ch);
 ```
 
 Request all rankings by date and by country. This call returns ranking records with all the available meta-data from the store's ranking list page.
+Currently the following countries are supported:
```
Comment (lknix) [inline source/includes/rankings.md]: Change to: The following countries are supported

File: source/includes/app-availability.md [hunk 4]
```diff
@@ -138,7 +138,23 @@ curl_close($ch);
 {"app_id":"com.BLI.CrossTheRoad","countries":["FR","JP","US"]}
 {"app_id":"com.singlecase.app","countries":["FR","DE","JP","US"]}
 ```
-Get a list of all the countries where the app is available.
+Get a list of all the countries where the app is available. Currently the following countries are supported:
```
Comment (lknix) [inline source/includes/app-availability.md]: Drop "Currently".

File: source/includes/reviews-for-all-apps.md [hunk 5]
```diff
@@ -178,9 +178,10 @@ Field                    | Description
 **app_version**          | The app version this review is for, if present. ![itunes_only](../images/itunes_logo.jpg)
 **date**                 | Review date as a string in ISO format: `YYYY-MM-DD`.
 **date_str**             | The original review date format, a string. ![android_only](../images/android_logo.jpg)
+**id**                   | The globally unique reviews ID. For iOS apps, we use IDs provided by App Store and for Android apps we generate them on our end by calculating a `md5_hash` on the following fields: `app_id`, `review_text`, `user_name`, `rating`, `title`, `language`.
```
Comment (lknix) [inline source/includes/reviews-for-all-apps.md]: Please refer to iOS/iTunes apps as "itunes apps" and for Android apps as "android apps". This is consistent with our `store=itunes|android` values.

Comment (lknix) [reply source/includes/reviews-for-all-apps.md]: Make sentences shorter & a few typos: For `itunes` apps, we use ID provided by App Store. For `android` apps we generate them by calculating MD5 hash from: `app_id`, `review_text`, `user_name`, `rating`, `title`, `language`.

File: source/includes/details-for-a-single-app.md [hunk 6]
```diff
@@ -323,6 +325,7 @@ Field                    | Description
 **price**                 | The price of the app, as displayed on the store side.
 **price_currency**        | Price currency, according to ISO 4271 standard.
 **price_value**           | Price value, expressed as digits only.
+**privacy_url**           | Privacy policy url, shown on store for the publisher of this app, if present. ![android_only](../images/android_logo.jpg)
```
Comment (lknix) [inline source/includes/details-for-a-single-app.md]: Privacy policy URL provided by the publisher, if present.

**Comment by lknix** [comment]
OK, a couple of comments but otherwise it looks good!

File: source/includes/app-publishers.md (hunk 1 above)
Comment (catalinstoica) [reply source/includes/app-publishers.md]: `date` is valid and required for app publishers api call:

```
$ curl --compressed -u "cata250907e3cd959e1cf0ea07beb3a3ee5f1348:X" "https://api.appmonsta.com/v1/stores/android/publishers.json"
{"message": "date is a required query parameter."}
```

Comment (catalinstoica) [reply source/includes/app-publishers.md]: Will be going through all the code samples and make sure they work!

File: source/includes/rankings.md (hunk 3 above)
Comment (catalinstoica) [reply source/includes/rankings.md]: Done.

File: source/includes/app-availability.md (hunk 4 above)
Comment (catalinstoica) [reply source/includes/app-availability.md]: Same.

File: source/includes/details-for-a-single-app.md (hunk 6 above)
Comment (catalinstoica) [reply source/includes/details-for-a-single-app.md]: https://github.com/AppMonsta/slate/commit/629db906d323a1abae3c0b64568ed9443ed91e2d

File: source/includes/reviews-for-all-apps.md (hunk 5 above)
Comment (catalinstoica) [reply source/includes/reviews-for-all-apps.md]: https://github.com/AppMonsta/slate/commit/ddd2a96deb6f898a62b2009a46485e063bf35701

File: source/includes/app-availability.md (hunk 4 above)
Comment (catalinstoica) [reply source/includes/app-availability.md]: https://github.com/AppMonsta/slate/commit/6a6e795fd366016d1a09c83c5dec646f981324c3

File: source/includes/rankings.md (hunk 3 above)
Comment (catalinstoica) [reply source/includes/rankings.md]: https://github.com/AppMonsta/slate/commit/6a6e795fd366016d1a09c83c5dec646f981324c3

File: source/includes/app-publishers.md (hunk 1 above)
Comment (catalinstoica) [reply source/includes/app-publishers.md]: https://github.com/AppMonsta/slate/commit/6a6e795fd366016d1a09c83c5dec646f981324c3

**Comment by lknix** [comment]
LGTM

**Pull request merged**